docs/book/v3/quick-start.md

```diff
@@ -2,7 +2,7 @@
 
 The laminas-hydrator component provides functionality for hydrating objects (which is the act of populating an object from a set of data) and extracting data from them.
 
-The component contains [concrete implementations](#available_implementations) for a number of common use cases, such as by using arrays, object methods, and reflection, and provides [interfaces](#base_interfaces) for creating custom implementations.
+The component contains concrete implementations for a number of common use cases, such as by using arrays, object methods, and reflection, and provides interfaces for creating custom implementations.
```

Comment:
The first link can remain, but needs to be corrected:

```markdown
[concrete implementations](#available-implementations)
```

https://docs.laminas.dev/laminas-hydrator/v4/quick-start/#available-implementations

Reply:
Done